test/extended/cli/debug.go

```diff
@@ -22,7 +27,7 @@ var _ = g.Describe("[sig-cli] oc debug", func() {
 	g.It("deployment configs from a build", func() {
 		err := oc.Run("create").Args("-f", testCLIDebug).Execute()
 		o.Expect(err).NotTo(o.HaveOccurred())
-		err = wait.Poll(cliInterval, cliTimeout, func() (bool, error) {
+		err = wait.Poll(cliInterval, buildTimeout, func() (bool, error) {
```

Comment:
This is a 5x bump from [the old 1m][1], which seems plenty generous.

[1]: https://github.com/openshift/origin/blob/2c51df8035383740abd5784e8c71244094797130/test/extended/cli/admin.go#L23